sycl/unittests/xpti_trace/QueueIDCheck.cpp

```diff
@@ -0,0 +1,189 @@
+//==------------ NodeCreation.cpp --- XPTI integration unit tests ----------==//
```

Comment:
```suggestion
//==------------ QueueIDCheck.cpp --- XPTI integration unit tests ----------==//
```
Not blocking

Reply:
fixing here https://github.com/intel/llvm/pull/11690